survive-the-monsters.js

```diff
@@ -117,211 +106,38 @@ function updateInput() {
     direction[1] = manY;
 }
```

Comment:
**nitpick**: Add a newline at the end of the file to adhere to POSIX standards.

Reply:
can you not nitpick about that? or at least provide some explanation of why im supposed to follow `Portable Operating System Interface` set of standards in javascript browser game? im not interested in creating an operating system or porting it